test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Date.Tests.ps1

```diff
@@ -277,3 +277,10 @@ Describe "Get-Date -UFormat tests" -Tags "CI" {
             Get-Date -Date $date -UFormat $format | Should -BeExactly $result
     }
 }
+
+Describe "Get-Date -FromUnixTime tests" -Tags "CI" {
```

Comment:
The `Describe` is no needed. You could put the test in first or second `Describe` in the file.

Reply:
Makes sense, I've updated the tests!